.github/workflows/cd-dev.yml

```diff
@@ -0,0 +1,57 @@
+name: Level1B Lambda CD (Dev)
```

Comment:
This workflow is mostly copied from `cd.yml`, but sets an extra environment variable and does _not_ lock branch to `deploy`. This could be accomplished using interactive variables in the original workflow instead, but since there are at the moment exactly two cases, I thought this was OK. Happy to change if you disagree, however.